packages/theme-chalk/src/common/var.scss

```diff
@@ -10,14 +10,29 @@
 
 /* Transition
 -------------------------- */
-$--all-transition: all 0.3s cubic-bezier(0.645, 0.045, 0.355, 1) !default;
-$--fade-transition: opacity 300ms cubic-bezier(0.23, 1, 0.32, 1) !default;
-$--fade-linear-transition: opacity 200ms linear !default;
-$--md-fade-transition: transform 300ms cubic-bezier(0.23, 1, 0.32, 1),
-  opacity 300ms cubic-bezier(0.23, 1, 0.32, 1) !default;
-$--border-transition-base: border-color 0.2s
-  cubic-bezier(0.645, 0.045, 0.355, 1) !default;
-$--color-transition-base: color 0.2s cubic-bezier(0.645, 0.045, 0.355, 1) !default;
+// refer to this website to get the bezier motion function detail
+//  https://cubic-bezier.com/#p1,p2,p3,p4 (change px as your function parameter)
+
+:root {
+
+  --el-transition-duration: 0.3s;
+  --el-fast-transition-duration: 0.2s;
```

Comment:
emm, I see that the naming rules for some variables usually put the `type: success/warning` behind the `kind: color/button`.

Should we use `--el-transition-duration-fast` instead of `--el-fast-transition-duration`?

Reply:
Nice catch, I'll update the variable name